src/newSeed/stage5.html

```diff
@@ -1,5 +1,51 @@
 <template>
-  <div class="page animated-page stageContainer stage5 au-animate ">
-    Stage ${stageNumber}
+  <div class="page animated-page stageContainer stage5 au-animate">
+    <div class="infoContainer">
+      <div class="heading heading3 title">Contact Details</div>
+      <div class="subtext">
+        Please provide your contact details and project logo.
+      </div>
+      <div>
+        <div class="infoInput">
+          <label for="email">Contact Email</label>
+          <input id="email" type="email" placeholder="name@mail.com"
+            value.bind="seedConfig.contactDetails.contactEmail"/>
+        </div>
+        <div class="infoInput">
+          <div class="labelContainer">
+            <label for="remarks">Additional Remarks</label>
+            <span value.from-view="seedConfig.contactDetails.remarks">
+              ${seedConfig.contactDetails.remarks.length}/250
+            </span>
+          </div>
+          <textarea
+              id="remarks"
+              value.bind="seedConfig.contactDetails.remarks"
+              maxlength="250"
+          ></textarea>
+        </div>
+        <div class="infoInput logo">
+          <div>
+            <label for="logo">Project Logo</label>
+            <input id="logo" type="text" placeholder="Valid Uri to project logo"
```

Comment:
Make placeholder:  "Valid Uri to the project's logo"

Reply:
Okay, English isn't my first language 😂